neuron/src/lib/Neuron/Zettelkasten/Query/Graph.hs

```diff
@@ -15,12 +15,15 @@ import Data.Dependent.Sum.Orphans ()
 import Data.GADT.Compare.TH
 import Data.GADT.Show.TH
 import Neuron.Zettelkasten.Graph.Type
+import Neuron.Zettelkasten.ID
 import Relude
 
 -- | Like `GraphQuery` but focused on the relationship between zettels.
 data GraphQuery r where
   -- | Query the entire graph.
   GraphQuery_Id :: GraphQuery ZettelGraph
+  -- | Query backlinks.
+  GraphQuery_BacklinksOf :: ZettelID -> GraphQuery ZettelGraph
```

Comment:
This should probably take `Connection` as another argument and return `[Zettel]` instead. You can use the [`backlinks`](https://github.com/srid/neuron/blob/ac46252e0d68a1d7176c223515b04a56f9e5cfa9/neuron/src/lib/Neuron/Zettelkasten/Graph.hs#L48-L52) function to run the query.